server/schedule/healthy.go

```diff
@@ -34,10 +35,8 @@ func IsRegionHealthyAllowPending(region *core.RegionInfo) bool {
 // rules is enabled, its peers should fit corresponding rules. When placement
 // rules is disabled, it should have enough replicas and no any learner peer.
 func IsRegionReplicated(cluster Cluster, region *core.RegionInfo) bool {
-	if cluster.GetOpts().IsPlacementRulesEnabled() {
-		return cluster.GetRuleManager().FitRegion(cluster, region).IsSatisfied()
-	}
-	return len(region.GetLearners()) == 0 && len(region.GetPeers()) == cluster.GetOpts().GetMaxReplicas()
+	filter := filter.NewRegionReplicatedFilter("option", cluster)
```

Comment:
ditto

Reply:
I think Replicated is a more complex logic than PendingPeer or DownPeer, so I only want to maintain it in one place